nbbuild/antsrc/org/netbeans/nbbuild/ParseProjectXml.java

```diff
@@ -854,7 +854,9 @@ private void translateModuleAutoDeps(String myCNB, List<Dep> deps, List<URL> mod
             }
             File jar = entry.getJar();
             if (!jar.isFile()) {
-                log("Cannot translate according to " + moduleAutoDeps + " because could not find " + jar, Project.MSG_WARN);
+                if (!Boolean.valueOf(getProject().getProperty("nbbuild.nowarn.missing.autodeps"))) {
+                    log("Cannot translate according to " + moduleAutoDeps + " because could not find " + jar, Project.MSG_WARN);
```

Comment:
The value of the property could influence severity - e.g. the message would always be logged, but (for example) with `MSG_DEBUG` when doing `ant clean`.

Reply:
Good point: what about interpreting the property's value as either true/false (disable/enable) or loglevel enum name ?